src/main/drivers/pwm_output.h

```diff
@@ -22,15 +22,14 @@
 #include "drivers/timer.h"
 
 typedef enum {
-    PWM_TYPE_STANDARD = 0,
+    PWM_TYPE_STANDARD,
     PWM_TYPE_ONESHOT125,
     PWM_TYPE_ONESHOT42,
     PWM_TYPE_MULTISHOT,
     PWM_TYPE_BRUSHED,
     PWM_TYPE_DSHOT600,
     PWM_TYPE_DSHOT300,
     PWM_TYPE_DSHOT150,
-    PWM_TYPE_MAX
```

Comment:
PWM_TYPE_MAX is also used
https://github.com/AlienWiiBF/betaflight/blob/61a3e91afc646090a2b8db445178af9f29fdb5c0/src/main/fc/fc_msp.c#L1445